assets/quick-start-module.js

```diff
@@ -324,7 +330,7 @@ function commandMessage(key) {
       "pip install --pre torch torchvision torchaudio -f https://download.pytorch.org/whl/nightly/cpu/torch_nightly.html<br /># On MacOS, we provide CPU-only packages, CUDA functionality is not provided",
 
     "preview,pip,macos,rocm4.0,python":
-      "pip install --pre torch torchvision torchaudio -f https://download.pytorch.org/whl/nightly/cpu/torch_nightly.html<br /># ROCm is not available on MacOS",
+      "<b>NOTE:</b> ROCm is not available on MacOS",
```

Comment:
This change is not necessary, as it follows the common pattern for MacOS platform, where all instruction point to CPU only package and in the comment mention that compute platform is not available

Reply:
Done